# goal
string MessageID
---
# result
string MessageResult
---
# feedback message
string MessageFeedback
